src/core_ghidra.cpp

```diff
@@ -132,109 +133,120 @@ static void ApplyPrintCConfig(RConfig *cfg, PrintC *print_c)
 	print_c->setMaxLineSize(cfg_var_linelen.GetInt(cfg));
 }
 
-static void Decompile(RCore *core, DecompileMode mode)
-{
-	DecompilerLock lock;
-
-#ifndef DEBUG_EXCEPTIONS
+static void refactored_decompile(RCore *&core, std::stringstream &out_stream,
+								DecompileMode mode, RAnnotatedCode *&code, ut64 addr){
```

Comment:
Excessive indent, you can keep this all on a single line.

Reply:
Changed as requested above.